Week1/homework/js-exercises/ex9.js

```diff
@@ -0,0 +1,13 @@
+//Exercise 9: Log the remainder
+//x after the calculation
+let x = 7;
+let calculationX = x % 3;
+console.log(`this is the remainder from 7/3 ➽ ${calculationX}(left over)`);
+//y after the calculation
+let y = 21;
+let calculationY = y % 4;
+console.log(`this is the remainder from 21/4 ➽ ${calculationY}(left over)`);
+//z after the calculation
+let Z = 13;
```

Comment:
why is `Z` a capital letter here?

Reply:
sorry that was typing, because i did my exercises in one JS file first and i had one small z before, i was more focusing about learning git with add and commit thats why i make them in one file and then did every ex in apart file.changes added.